src/Generator/JoeGenerator.jl

```diff
@@ -73,3 +73,43 @@ function τ⁻¹(::Type{T},tau) where T<:JoeGenerator
         return Roots.find_zero(θ -> _joe_tau(θ) - tau, (one(tau),tau*Inf))
     end
 end
+
+function _rho_joe_via_cdf(θ; rtol=1e-7, atol=1e-9, maxevals=10^6)

```

Comment:
So this rho is valid only for d=2 or for other dimensions as well ? Then the rho_inv fuction below is valid for d=2 only or for other dimensions as well ?